packages/core/template/portfolio/contents/project2.md

```diff
@@ -0,0 +1,23 @@
+<frontmatter>
+  title: Project 2
+</frontmatter>
+
+<br>
```

Comment:
Deleting this linebreak and the one in project 2 does not seem to have any effect, wld suggest removing them